packages/create-react-app/index.js

```diff
@@ -105,7 +105,7 @@ function createApp(name, verbose, version) {
   checkAppName(appName);
 
   if (!pathExists.sync(name)) {
-    fs.mkdirSync(root);
```

Comment:
Let's just use the [`mkdirp` package](https://www.npmjs.com/package/mkdirp), because we only need this one thing, it's small and `fs.extra` would just include `mkdirp` as a dependency.

We'll be able to get rid of `pathExists` so the total number of dependencies is still the same :) Checking if a folder/file exists before creating it [is an anti-pattern](https://github.com/sindresorhus/path-exists/blob/master/readme.md#path-exists-), so trying to create the directory and just checking if it existed is also better in that sense.